src/display/font_loader.js

```diff
@@ -77,6 +77,22 @@ var FontLoader = {
     ));
   },
 
+  get isEvalSupported() {
+    var evalSupport;
+    if (PDFJS.isEvalSupported) {
```

Comment:
define PDFJS.isEvalSupported in the dispplay/api.js and make it `true` by default.